src/components/BaseComponents/BaseMap.tsx

```diff
@@ -1,10 +1,12 @@
 import { MapContainer, TileLayer } from "react-leaflet";
-import StationsContext from "@/context/stations";
-import { useContext } from "react";
+import { MapLeafletType } from "@/types";
+import configuration from "../../app/appConfig";
 
-export default function BaseMap(props: mapLeaflet) {
-    const { appVersion } = useContext(StationsContext);
-    const attribution = process.env.NODE_ENV === "development" ? `| App version: ${appVersion}` : "";
+export default function BaseMap(props: Readonly<MapLeafletType>) {
```

Comment:
Sonar was complaining because of this rule https://rules.sonarsource.com/typescript/RSPEC-6759/ and suggests to enforce it where possible.

Reply:
Yeah, it makes sense. Although we don't mutate props in the child, let's keep it.

Reply:
There'll be more changes and additions in the types soon, so I tried to keep the changes to the types here minimal to avoid increaing the scope of the PR.